manifests/concourse-manifest/concourse-base.yml

```diff
@@ -99,6 +102,7 @@ jobs:
         forward_host: 127.0.0.1
         atc:
           address: 127.0.0.1:8080
+          old_resource_grace_period: 180m
```

Comment:
This [option is documented as](http://bosh.io/jobs/atc?source=github.com/concourse/concourse&version=1.1.0#p=old_resource_grace_period):

> old_resource_grace_period: 5m 
> How long to cache the result of a get step after a newer version of the resource is found. Use Go duration format (1m = 1 minute).

Is raising this going to have any negative effects? Do it need to be so high?